examples/quick_start/plot_quick_start.py

```diff
@@ -0,0 +1,289 @@
+"""
+.. _quick start:
+
+Quick Start
+===========
+
+The purpose of this hands-on tutorial is to acquaint first-time users to all the basic components
+of a starfish pipeline. Anyone with a computer and internet connection should be able to follow
+the guide to create and run the example starfish pipeline on the provided experiment data.
+
+Approximate time to complete tutorial: 10 minutes
+
+Prerequisites
+-------------
+
+* Python 3.6+ installed (Python 3.7 recommended)
+* Some basic knowledge of scientific computing in Python_
+* :ref:`Starfish library installed <installation>`
+
+.. _Python: https://docs.scipy.org/doc/numpy/user/quickstart.html
+
+Starfish Pipeline
+-----------------
+
+Open a Jupyter notebook or console with IPython. Make sure you're using the kernel or virtual
+environment where you installed starfish, and then import the following modules:
+"""
+
+from starfish import data
+from starfish.image import Filter
+from starfish.image import ApplyTransform, LearnTransform
+from starfish.spots import FindSpots, DecodeSpots
+from starfish.types import Features, Axes
+from starfish.image import Segment
+from starfish.spots import AssignTargets
+from starfish import ExpressionMatrix
+from starfish import ImageStack
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
```

Comment:
for tutorials, I prefer to do imports in the same block that they are used. a large list of imports is likely to be glanced over. keeping the imports in the same block where they are used lets the user understand where each processing step lives.